src/components/plans/index/PlanIndex.js

```diff
@@ -1,6 +1,7 @@
 import React, {Component, PropTypes} from 'react'
```

Comment:
Maybe there should be some callout at the top of the page if they are logged out that says 'log in to select a plan' just so viewers know they have to be signed in.